overview/configuration-options.md

```diff
@@ -4,129 +4,47 @@
-    </tr>
-    <tr>
-      <td style="text-align:left">nonExplicitSupportedLngs</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">if true will pass eg. <code>en-US</code> if finding <code>en</code> in supportedLngs</td>
-    </tr>
-    <tr>
-      <td style="text-align:left">load</td>
-      <td style="text-align:left">&apos;all&apos;</td>
-      <td style="text-align:left">language codes to lookup, given set language is &apos;en-US&apos;: <code>&apos;all&apos; --&gt; [&apos;en-US&apos;, &apos;en&apos;, &apos;dev&apos;]</code>, <code>&apos;currentOnly&apos; --&gt; &apos;en-US&apos;</code>, <code>&apos;languageOnly&apos; --&gt; &apos;en&apos;</code>
-      </td>
-    </tr>
-    <tr>
-      <td style="text-align:left">preload</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">array of languages to preload. Important on serverside to assert translations
-        are loaded before rendering views.</td>
-    </tr>
-    <tr>
-      <td style="text-align:left">lowerCaseLng</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">locale will be fully lowercased eg. <code>en-US</code> --&gt; <code>en-us</code>
-      </td>
-    </tr>
-    <tr>
-      <td style="text-align:left">cleanCode</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">language will be lowercased <code>EN</code> --&gt; <code>en</code> while leaving
-        full locales like <code>en-US</code>
-      </td>
-    </tr>
-    <tr>
-      <td style="text-align:left">ns</td>
-      <td style="text-align:left">&apos;translation&apos;</td>
-      <td style="text-align:left">string or array of namespaces to load</td>
-    </tr>
-    <tr>
-      <td style="text-align:left">defaultNS</td>
-      <td style="text-align:left">&apos;translation&apos;</td>
-      <td style="text-align:left">default namespace used if not passed to <a href="../translation-function/essentials.md">translation function</a>
-      </td>
-    </tr>
-    <tr>
-      <td style="text-align:left">fallbackNS</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">string or array of namespaces to lookup key if not found in given namespace.
-        <a
-        href="../principles/fallback.md#namespace-fallback">Learn more</a>.</td>
-    </tr>
-    <tr>
-      <td style="text-align:left">partialBundledLanguages</td>
-      <td style="text-align:left">false</td>
-      <td style="text-align:left">allows some resources to be set on initialization while others can be
-        loaded using a backend connector</td>
-    </tr>
-  </tbody>
-</table>
-
-### missing keys
+### Languages, namespaces, resources
+|option|default|description|
+| :--- | :--- | :--- |
+| resources | undefined | resources to initialize with (if not using loading or not appending using [`addResourceBundle`](../overview/api#addresourcebundle))|
```

Comment:
shouldn't the link be like this? `api.md#addresourcebundle`

Reply:
IIRC I just copied the format from the same file, or maybe some other file... Since I don't have the gitbook running locally, I wasn't able to test what works best, sorry :upside_down_face:

Reply:
lol right, that was an oversight